cmd/network/verification.go

```diff
@@ -87,12 +89,13 @@ func NewCmdValidateEgress() *cobra.Command {
 	}
 
 	validateEgressCmd.Flags().StringVar(&e.ClusterId, "cluster-id", "", "(optional) OCM internal/external cluster id to run osd-network-verifier against.")
-	validateEgressCmd.Flags().StringVar(&e.SubnetId, "subnet-id", "", "(optional) private subnet ID override, required if not specifying --cluster-id")
+	validateEgressCmd.Flags().StringArrayVar(&e.SubnetId, "subnet-id", nil, "(optional) private subnet ID override, required if not specifying --cluster-id")
```

Comment:
```suggestion
	validateEgressCmd.Flags().StringArrayVar(&e.SubnetId, "subnet-id", nil, "(optional) private subnet ID override, required if not specifying --cluster-id" and can be specified multiple times to run against multiple subnets)
```
We should add some documentation around the new functionality. I think an example would be nice too above, around lines 69-85